examples/resources/snowflake_shared_database/resource.tf

```diff
@@ -0,0 +1,11 @@
+resource "snowflake_shared_database" "test" {
+  name                  = "shared_database"
+  from_share            = "organization_name.account_name.share_name"
```

Comment:
Why are data_retention_time_in_days and max_data_extension_time_in_days missing?

Reply:
Because you cannot set those values on shared database (they're not in schema).